### PR TITLE
+slf #16630 Add the actor system name to the Slf4jLogger context

### PR DESCRIPTION
**Pull request by meln1k**

### CODE REVIEW

**Comment by ktoso** [comment]
OK TO TEST

**Comment by ktoso** [comment]
refs #16630

**Comment by meln1k** [comment]
@ktoso Seems that I have a wrong branch name prefix. This branch is ready to be merged, however wip prefix says it's not ready. Should I make a new PR with a valid branch name?

**Comment by ktoso** [comment]
Hello @meln1k, the branch name is OK, no worries about it - what we really care about is the commit message (1st line mostly), which in your case is exactly as we need it :-)

We'll merge soon, needs a quick look (*LooksGoodToMe*'s) from a few team members before that.

File: akka-docs/rst/java/logging.rst [hunk 1]
```diff
@@ -305,7 +305,7 @@ the first case and ``LoggerFactory.getLogger(String s)`` in the second).
 
   final LoggingAdapter log = Logging.getLogger(system.eventStream(), "my.string");
 
-Logging Thread and Akka Source in MDC
+Logging Thread, Akka Source and Actor System in MDC
```
Comment (ktoso) [inline akka-docs/rst/java/logging.rst]: Maybe I'm a bit OCD about those things but aligning the `---` with the title is usually a nice thing to do :)

**Comment by ktoso** [comment]
Thanks for the contribution - LGTM. Would like one more hakker to review this before merge. :)

**Comment by patriknw** [comment]
LGTM
Thanks for a great contribution, @meln1k

**Pull request merged**